src/staging/src/components/search/mainSearch.js

```diff
@@ -1,31 +1,26 @@
 import React, { useState } from "react"
 import PropTypes from "prop-types"
-import { TextField, Button, Link, InputAdornment } from "@material-ui/core"
+import { TextField, Button, InputAdornment, InputLabel } from "@material-ui/core"
 import "../../styles/search.scss"
 import { navigate } from "gatsby"
-import SearchFilter from "./searchFilter"
+import SearchIcon from "@material-ui/icons/Search"
 
-const MainSearch = ({ data: { activities, facilities } }) => {
-  const activityItems = activities.map(a => ({
-    label: a.activityName,
-    value: a.activityNumber,
-  }))
+  
+const MainSearch = () => {
 
-  const facilityItems = facilities.map(f => ({
-    label: f.facilityName,
-    value: f.facilityNumber,
-  }))
-  const [openFilter, setOpenFilter] = useState(false)
-  const [quickSearch, setQuickSearch] = useState({
+  // search options not set here, but need to be passed to search page
+  // via state, so setting all to initial values below
```

Comment:
You can remove quickSearch, activities, etc and just call navigate without them:

```javascript
    navigate("/explore", {
      state: {
        searchText,
      },
    })
```

Reply:
Done. Also updated index to not pass all activities, facilities and parks as they are not necessary.